api/datasets/blackspots/exporter.py

```diff
@@ -0,0 +1,31 @@
+from django.db import transaction
+
+from datasets.blackspots.models import Spot, SpotExport
+
+
+class SpotExporter:
+
+    @staticmethod
+    def update_export_table():
+        """
+        Update the Spots export table
+        """
+        with transaction.atomic():
```

Comment:
Het gaat om ruim 300 spots op dit moment, dus een setup als hieronder is wel te doen qua performance.